nilearn/datasets/struct.py

```diff
@@ -152,7 +152,8 @@ def load_mni152_template(resolution=None):
 
     if resolution is None:
         warnings.warn("Default resolution of the MNI template will change "
-                      "from 2mm to 1mm in version 0.10.0", FutureWarning)
+                      "from 2mm to 1mm in version 0.10.0", FutureWarning,
+                      stacklevel=2)
```

Comment:
why stacklevel=2? this means specify the caller as the function from which the warning was issued

Reply:
I wanted to solve the issue of having part of the warning message repeated in the log, as you can see from this example:

```
/home/analu/mygit/nilearn/nilearn/datasets/struct.py:154: FutureWarning: Default resolution of the MNI template will change from 2mm to 1mm in version 0.10.0
  warnings.warn("Default resolution of the MNI template will change "
```

Is there any other way to solve this?